tests/disabled_user_test.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""Test auth_role.DISABLE and api_perm.require_not_disabled
+
+:copyright: Copyright (c) 2024 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import pytest
+
+
+def test_disabled_user(fc):
+    from pykern.pkcollections import PKDict
+    from pykern.pkunit import pkeq, pkok
+    import sirepo.auth_role
+    import sirepo.pkcli.roles
+
+    a = fc.sr_auth_state()
+    pkok(
+        sirepo.auth_role.ROLE_DISABLED not in fc.sr_auth_state().roles,
+        "{} found in roles",
+        sirepo.auth_role.ROLE_DISABLED,
+    )
+    fc.sr_post(
```

Comment:
This should test a few APIs, e.g. running a simulation. You'll need to cache the request info for this case so that the request goes straight to the uri.